tool/update_version.dart

```diff
@@ -0,0 +1,388 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
why was this file moved out of commands?

Reply:
oh shoot this is an artifact I forgot to push up my remove